ramp.yml

```diff
@@ -32,7 +32,14 @@ exclude_commands:
     - rg.pyfreeinterpreter
 overide_command:
     - {"command_arity": -1, "command_name": "rg.trigger", "first_key": 2, "flags": ["readonly"], "last_key": 2, "step": 1}
+add_command:
+    - {"command_arity": -1, "command_name": "rg.jexecute", "first_key": 0, "flags": ["readonly"], "last_key": 0, "step": -1}
+    - {"command_arity": -1, "command_name": "rg.vec_sim", "first_key": 0, "flags": ["readonly"], "last_key": 0, "step": 0}
+    - {"command_arity": -1, "command_name": "rg.vec_add", "first_key": 1, "flags": ["write", "deny-oom"], "last_key": 1, "step": 1}
 dependencies:
     {{GEARS_PYTHON3_NAME}}:
         url: http://redismodules.s3.amazonaws.com/redisgears/{{GEARS_PYTHON3_FNAME}}
         sha256: {{GEARS_PYTHON3_SHA256}}
+    gears_jvm:
+        url: http://redismodules.s3.amazonaws.com/redisgears_plugins/jvm_plugin/snapshots/gears-jvm.{{OS_DESC}}.master.tgz
+        sha256: 76e0c18751da724cc16acbe2be04f47a66e90f9efad81247afde8d5d07e5928a
```

Comment:
We still need to care of the sha256 generation for `gears-jvm*.tgz` in `pack.sh`.